### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [OpenTracksApp/OpenTracks#992] [General]
Would a Mastodon account for this project be possible?

**Comment by dennisguse** [comment]
Definitely is this possible.
From my side, it is just a lack of time (and to some degree lack of interest in social media 🤷‍♂️ ).

Sometimes, we post from our personal accounts with #OpenTracks: https://chaos.social/tags/opentracks

Long story short: it is a great idea!
It is just a matter of doing it. 
@w4ts0n would have / interest to support us in this?

@pstorch @rgmf @mondlicht-und-sterne any other ideas or a veto against this?

**Comment by pstorch** [comment]
If someone has the time to manage such an account, I'm fine with it.

**Reply by linux-lukas** [reply]
In principle, I would be able to help you run the Mastodon account. 

I don't know if there is already an automated way to mirror release entries to Mastodon, that could already serve the main purpose and thus reduce the time spent. But unfortunately I couldn't program it either.😅 
@dennisguse @pstorch

**Reply by mondlicht-und-sterne** [reply]
> If someone has the time to manage such an account, I'm fine with it.

but that would have to be done by a programmer. i can't imagine managing it, but i think it's a good idea.

**Reply by pstorch** [reply]
@Lukas2112 

> an automated way to mirror release entries to Mastodon

Would that be the primary expectation of such an account? What about answering user questions? That would be more time consuming.

**Reply by dennisguse** [reply]
Releases infos are available via RSS: https://github.com/opentracksapp/opentracks/releases.atom

What about a bot that posts every release and reposts every #OpenTracks tagged post?

Some more content could be also shared.

**Reply by mondlicht-und-sterne** [reply]
> Releases infos are available via RSS: https://github.com/opentracksapp/opentracks/releases.atom
> 
> What about a bot that posts every release and reposts every #OpenTracks tagged post?
> 
> Some more content could be also shared.

like this for the feedreader ;)

https://chaos.social/@dennisguse.rss

**Reply by dennisguse** [reply]
What about something like this?

https://github.com/marketplace/actions/toot-together

**Reply by dennisguse** [reply]
We add this link to the webpage: https://chaos.social/tags/OpenTracks
(also has RSS support)
